docs/maintain/elasticsearch.rst

```diff
@@ -124,8 +111,12 @@ If the `ReadOnlyREST plugin <https://readonlyrest.com>`__ is used:
 
       ./run.py 'docs' cmd.run "/usr/share/elasticsearch/bin/elasticsearch-plugin remove readonlyrest"
 
-#. Update ``readonlyrest_version`` and ``source_hash`` in the ``salt/elasticsearch/plugins/readonlyrest.sls`` file
-
 #. :doc:`Deploy the service<../deploy/deploy>`
 
+#. Test the `site search works correctly <https://standard.open-contracting.org/latest/en/search/?q=example&check_keywords=yes&area=default`__.
+
+.. note::
+
+   A new version might not yet be available for download. You can check the `changelog <https://github.com/beshu-tech/readonlyrest-docs/blob/master/changelog.md>`__.
```

Comment:
Should this instead be a first step, in order to get the appropriate values for readonlyrest_version and elasticsearch_version?

Reply:
Agreed, updated in new commit.